workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFRequestList/utils/getMDFListColumns.tsx

```diff
@@ -83,6 +86,47 @@ export default function getMDFListColumns(
 					});
 				}
 
+				if (currentValue === PermissionActionType.CANCEL && row[MDFColumnKey.STATUS] === Status.APPROVED.name) {
+					const {data: mdfRequest} = useGet<
+						MDFRequestDTO
+					>(
+						row[MDFColumnKey.ID] &&
+							`/o/${LiferayAPIs.OBJECT}/${ResourceName.MDF_REQUEST_DXP}/${row[MDFColumnKey.ID]}?nestedFields=mdfReqToActs%2C%20mdfReqToMDFClms`
+					);
+					previousValue.push({
+						icon: 'block',
+						key: Status.CANCELED.key,
+						label: ' Cancel',
+						onClick: () => {
+							Liferay.Util.openConfirmModal({
+								message:
+									'Are you sure to cancel the MDF request?',
```

Comment:
I think this should read as "Are you sure you want to cancel the MDF request?". "Are you sure to cancel" isn't grammatically correct.

Reply:
hey Kyle! i've just used the text that is on the ticket. should i ask Aryan about this?

Reply:
Yes, please ask about it on the ticket then. Thanks!

Reply:
Hey @abfranca, you could ask to Caio Monteiro.

Reply:
Done!